### PR TITLE
Srl 4680 adjust issues and support install roles

### PR DESCRIPTION
**Pull request by Maycon-Santos-sm**
* Set full path for outputs
* Fixed default requirements path
* Conditional creation of ~/.netrc
* Conditional execution of ansible-galaxy to install roles
* Update documentation

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -43,13 +47,33 @@ runs:
       with:
         python-version: ${{ inputs.python-version }}
 
+    - id: check-req-input
+      shell: bash
+      run: |
+        if [[ "x-${{ inputs.python-requirements }}" == "x-not-set" ]]
```
Comment (andreas-zapantis) [inline action.yml:53]: Question for my understanding, but why do you use the `x-` prefixes here?

File: action.yml [hunk 2]
```diff
@@ -43,13 +47,33 @@ runs:
...
+            echo "::set-output name=req-path::$REQ_PATH"
+            SHA256=$(sha256sum $REQ_PATH |awk '{print $1}')
+        else
+            echo "::set-output name=req-path::$(echo ${{ inputs.python-requirements }})"
+            SHA256=$(sha256sum ${{ inputs.python-requirements }} |awk '{print $1}')
+        fi
+        echo "::set-output name=req-sha256::$SHA256"
+
+    - id: set-venv-dir
+      run: echo "::set-output name=dir::$(echo ${{ github.workspace }}/venvansible)"
```
Comment (andreas-zapantis) [inline action.yml:65]: Nice, i got this issue where this was not running when used in combination with `working-directory`.
This should fix it.

Comment (Maycon-Santos-sm) [reply action.yml:65]: Saw that and add a small test here:  https://github.com/searchmetrics/sm-ansible-action/pull/2/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R37

File: action.yml (hunk 1 above)
Comment (Maycon-Santos-sm) [reply action.yml:53]: little tricky to avoid the odd case of empty variable breaking the IF expression

**Review by andreas-zapantis — APPROVED** [review]

**Review by pmotwani — APPROVED** [review]

**Pull request merged**